### PR TITLE
h2s spec maven plugin 1.0.0 with h2spec 2.6.0

### PR DESCRIPTION
**Pull request by olamy**
See release notes https://github.com/jetty-project/h2spec-maven-plugin/releases/tag/h2spec-maven-plugin-1.0.0

### CODE REVIEW

File: jetty-http2/http2-server/pom.xml [hunk 1]
```diff
@@ -17,11 +17,15 @@
...
       <plugin>
-        <groupId>com.github.madgnome</groupId>
+        <groupId>org.mortbay.jetty</groupId>
         <artifactId>h2spec-maven-plugin</artifactId>
         <configuration>
           <mainClass>org.eclipse.jetty.http2.server.H2SpecServer</mainClass>
           <skip>${skipTests}</skip>
+          <junitPackage>org.eclipse.jetty.h2spec</junitPackage>
+          <excludeSpecs>
+            <excludeSpec>3.5 - Sends invalid connection preface</excludeSpec>
```
Comment (joakime) [inline jetty-http2/http2-server/pom.xml:28]: Is this right?
Are we actually excluding this specific test?
If so, perhaps we should include a `<!-- comment explaining why -->`?

File: jetty-http2/http2-server/pom.xml [hunk 2]
```diff
@@ -78,4 +82,34 @@
...
+            <artifactId>exec-maven-plugin</artifactId>
+            <executions>
+              <execution>
+                <id>spec-server-run</id>
+                <phase>test</phase>
+                <configuration>
+                  <mainClass>org.eclipse.jetty.http2.server.H2SpecServer</mainClass>
+                  <classpathScope>test</classpathScope>
+                  <arguments>
+                    <argument>8080</argument>
```
Comment (joakime) [inline jetty-http2/http2-server/pom.xml]: Pick a different port.
Port 8080 is often used by other processes on the same machine.
How about 28888 ?

Comment (olamy) [reply jetty-http2/http2-server/pom.xml]: this one is just for manual local test and activated only by a profile. The real test part of the build is using random port

File: jetty-http2/http2-server/pom.xml (hunk 1 above)
Comment (olamy) [reply jetty-http2/http2-server/pom.xml:28]: @lorban do you remember why we need to exclude this test?

Comment (lorban) [reply jetty-http2/http2-server/pom.xml:28]: Last time I worked on the h2spec failures, I remember that one test added to the not-yet-released version of H2Spec was constantly failing, but I did not investigate it. Do you want me to?

Comment (olamy) [reply jetty-http2/http2-server/pom.xml:28]: If you have time otherwise we only need a comment.

File: jetty-http2/http2-server/pom.xml (hunk 2 above)
Comment (olamy) [reply jetty-http2/http2-server/pom.xml]: done

**Pull request merged**